src/reducers/productReducer.js

```diff
@@ -0,0 +1,88 @@
+import { Actions } from "./Actions";
+import { appendItem, updateQuantity, removeItemFromCartList } from "./utils";
+
+const productReducer = (prevState, action) => {
```

Comment:
A really minor suggestion here. `action` can be destructured as `{type, payload}` here.  Will reduce effort of writing ` action.type` and `action.payload`